test/format/format_config.c

```diff
@@ -1449,6 +1451,90 @@ config_statistics(void)
 }
 
+/*
+ * config_chunk_cache --
+ *     Chunk cache configuration.
+ */
+static void
+config_chunk_cache(void)
+{
+    char buf[128];
+    const char *chunkcache_type;
+    const char *storage_source;
+
+    storage_source = GVS(TIERED_STORAGE_STORAGE_SOURCE);
+    chunkcache_type = NULL;
+
+    /* Chunkcache does not work unless tiered storage is configured with dir_store. */
+    if (!g.tiered_storage_config || strcmp(storage_source, "dir_store") != 0) {
+        if (config_explicit(NULL, "chunk_cache") && GV(CHUNK_CACHE))
+            testutil_die(EINVAL,
+              "%s: chunkcache cannot be enabled unless tiered storage is configured with dir_store",
+              progname);
+        return;
+    }
+
+    if (!config_explicit(NULL, "chunk_cache")) {
+        /*
+         * Make sure no configurations related to chunk caching are set if chunkcache is not
+         * enabled.
+         */
+        if (config_explicit(NULL, "chunk_cache.capacity") ||
+          config_explicit(NULL, "chunk_cache.chunk_size") ||
+          config_explicit(NULL, "chunk_cache.type") ||
+          config_explicit(NULL, "chunk_cache.storage_path"))
+            testutil_die(EINVAL,
+              "%s: Enable chunk caching (chunk_cache=on) to allow configuring other chunk cache "
+              "settings",
+              progname);
+
+        /* Enable chunkcache 50% of the time if not explicit set. */
+        testutil_snprintf(
+          buf, sizeof(buf), "chunk_cache=%s", mmrand(&g.data_rnd, 1, 100) <= 100 ? "on" : "off");
+        config_single(NULL, buf, false);
+    }
+
+    if (GV(CHUNK_CACHE)) {
+        if (config_explicit(NULL, "chunk_cache.type")) {
+            chunkcache_type = GVS(CHUNK_CACHE_TYPE);
+            if (strcmp(chunkcache_type, "FILE") != 0 && strcmp(chunkcache_type, "DRAM") != 0)
+                testutil_die(EINVAL, "illegal chunkcache.type configuration: %s", chunkcache_type);
+
+            if (GV(RUNS_IN_MEMORY) && strcmp(chunkcache_type, "FILE") == 0)
+                testutil_die(EINVAL,
+                  "%s: chunk caching cannot be enabled for in-memory runs as chunkcache.type is "
+                  "set to FILE.",
+                  progname);
+        } else {
+            if (GV(RUNS_IN_MEMORY))
+                config_single(NULL, "chunkcache.type=DRAM", false);
+            else {
+                /*
+                 * Alternate between running chunk cache with the 'File' type and the 'DRAM' type.
+                 */
+                testutil_snprintf(buf, sizeof(buf), "chunk_cache.type=%s",
+                  mmrand(&g.data_rnd, 1, 100) <= 50 ? "DRAM" : "FILE");
+                config_single(NULL, buf, false);
+            }
+        }
+
+        if (strcmp(GVS(CHUNK_CACHE_TYPE), "FILE") == 0 &&
+          !config_explicit(NULL, "chunk_cache.storage_path")) {
+            testutil_snprintf(buf, sizeof(buf), "chunk_cache.storage_path=/tmp/chunkcache%" PRIu32,
+              mmrand(&g.data_rnd, 1, 1000000000));
```

Comment:
I think we should use a name like "WiredTigerChunkCache" in the wiredtiger home directory.  Having it in tmp means tests may leave large remnant files.

Reply:
Also many `/tmp` file systems are simply in the root file system on instances and would likely hit ENOSPC pretty quickly. Also if there are bugs in chunk cache code, having it in the wiredtiger home directory as @ddanderson suggests means it will be part of the artifacts and everything in the directory is self-contained.

Reply:
That's a reasonable take. We had this originally, but having large binary files in every single saved Evergreen artifact seemed like a bad idea. If 2GB is fine, then it's easy to change this back to living in WT's home directory. We could combine this with a smaller cache.

Reply:
I have set the maximum chunkcache capacity to be (randomly) configured to less than 1GB (too less?) in format testing unless explicitly provided (in which case it can be configured up to 100GB).  

1GB seems reasonably sized to be in the DB directory. I will make this change.

Reply:
Updated in [1d223f2](https://github.com/wiredtiger/wiredtiger/pull/9589/commits/1d223f2e2a7b44043cd62433c9e8b51a69d4516d)